source/conf-netlify/conf.py

```diff
@@ -26,7 +26,7 @@
         "4.2": "https://docs.percona.com/percona-server-for-mongodb/4.2/",
         "4.4": "https://docs.percona.com/percona-server-for-mongodb/4.4/",
         "5.0": "https://docs.percona.com/percona-server-for-mongodb/5.0/",
```

Comment:
Should we then maybe change the 5.0 link as well?
```suggestion
        "5.0 (Release Candidate)": "https://docs.percona.com/percona-server-for-mongodb/5.0/",
```

Reply:
As this label occupies 2 lines and looks ugly, it's decided to leave 5.0 label as is. @LenzGr please confirm

Reply:
Fine with me!